### PR TITLE
DBZ-978 Added last event and captured tables metrics

### PR DESCRIPTION
**Pull request by jpechane**
Requires https://github.com/debezium/debezium/pull/704

### CODE REVIEW

File: debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerTaskContext.java [hunk 1]
```diff
@@ -15,7 +15,7 @@
  */
 public class SqlServerTaskContext extends CdcSourceTaskContext {
 
-    public SqlServerTaskContext(SqlServerConnectorConfig config) {
-        super("SQL Server", config.getLogicalName());
+    public SqlServerTaskContext(SqlServerConnectorConfig config, SqlServerDatabaseSchema schema) {
+        super("SQLServer", config.getLogicalName(), schema::tableIds);
```
Comment (gunnarmorling) [inline debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerTaskContext.java]: Is this the correct spelling? I find "SQL Server" (two words) more often on the web.

Comment (jpechane) [reply debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerTaskContext.java]: @gunnarmorling I originally used it too but it is not practical when it is used in JMX names and dashboard queries

Comment (gunnarmorling) [reply debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerTaskContext.java]: Ok, I get the reasoning. I'd suggest to rename it to "SQL_Server" then. I'll do that change and apply.

**Comment by gunnarmorling** [comment]
Rebased and applied. Thanks, @jpechane!

**Pull request merged**